source/overloading.tex

```diff
@@ -983,7 +982,7 @@
 String operator + (const String&, const String&);
 
 void f() {
- const char* p= "one" + "two";  // ill-formed because neither operand has class or enumeration type
+ const char* p= "one" + "two";  // error: neither operand has class or enumeration type
```

Comment:
// error: cannot add two pointers; overloaded `operator+` not considered
// because neither operand has class or enumeration type

Reply:
Fixed. (Also fixed code indentation.)